1-exercises/A-this/exercise-2.js

```diff
@@ -6,18 +6,20 @@
 
 */
 
-
 let person = {
-    name: "Maggie",
-    age: 26,
-    currentAddress: "London",
-    changeAddress: /* todo */
+  name: "Maggie",
+  age: 26,
+  currentAddress: "London",
+  changeAddress: function (newAddress) {
+    return (this.currentAddress = newAddress);
```

Comment:
Although no requirement on the returned value is specified. Its ok to return the updated address.